README.md

```diff
@@ -23,13 +23,13 @@ For Ethereum network data, you can either run your own Ethereum node or use an E
 
 ### Running a Local Graph Node
 
-This is a quick example to show a working Graph Node. It is a [subgraph for the Ethereum Name Service (ENS)](https://github.com/graphprotocol/ens-subgraph) that The Graph team built.
+This is a quick example to show a working Graph Node. It is an [example subgraph for Gravatars](https://github.com/graphprotocol/example-subgraph) that The Graph team built.
```

Comment:
We should no longer refer to a "The Graph team", lets just remove that part.

Reply:
Fixed!